site/content/docs/5.2/examples/cheatsheet-rtl/index.html

```diff
@@ -1389,26 +1389,26 @@ <h3>شريط التقدم</h3>
       <div>
         {{< example show_markup="false" >}}
         <div class="progress mb-3">
-          <div class="progress-bar" role="progressbar" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100">0%</div>
+          <div class="progress-bar" role="progressbar" aria-label="Example with label" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100">0%</div>
         </div>
         <div class="progress mb-3">
-          <div class="progress-bar bg-success w-25" role="progressbar" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100">25%</div>
+          <div class="progress-bar bg-success w-25" aria-label="Success example with label" role="progressbar" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100">25%</div>
```

Comment:
```suggestion
          <div class="progress-bar bg-success w-25" role="progressbar" aria-label="Success example with label" aria-valuenow="25" aria-valuemin="0" aria-valuemax="100">25%</div>
```
Was OK but just for consistency.